### PR TITLE
PP-499: Remove irrelevant code from pbs_postinstall script

### PR DESCRIPTION
**Pull request by nithinj**
#### Issue-ID

* **[PP-499](https://pbspro.atlassian.net/browse/PP-499)**

#### Problem description
* Code clean up of pbs_postinstall

#### Cause / Analysis
* Cleanup pbs_postinstall code by removing code for taking pgsql backup and correcting indentation.

#### Solution description
* Removing the lines of code which is taking a backup of pgsql
* Fixing indentation issues

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: src/cmds/scripts/pbs_postinstall.in [hunk 1]
```diff
@@ -99,31 +99,6 @@ createpath() {
...
-			[ ! -d "${2}/${dir}.${stamp}" ] && break
-			let i+=1
-		done
-	fi
-	echo "*** Backing up $1 to ${2}/${dir}.$stamp"
-	echo "*** ${2}/${dir}.$stamp may need to be manually removed"
-	cp -pr "$1" "${2}/${dir}.$stamp" 2>&1
-	return 0
-}
-
```
Comment (mike0042) [inline src/cmds/scripts/pbs_postinstall.in]: Bummer. I thought that was a nifty little function. I suppose we can always grab it from the initial OSS release if we need it in the future.

Comment (nithinj) [inline src/cmds/scripts/pbs_postinstall.in]: This piece of code is removed as we are not performing the backup activity any longer in pbs_postinstall.

**Comment by mike0042** [comment]
I restarted the build and it completed successfully this time.

**Comment by nithinj** [comment]
Thanks Mike for restarting the build. 

@PBSPro/pbspro-maintainers , Please proceed with the merging.

**Pull request merged**